app/views/user_email_mailer/response_invite.html.erb

```diff
@@ -3,215 +3,139 @@
 <head>
   <title>Email letter</title>
   <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
+  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1.0"/>
   <style>
-      @import url('https://fonts.googleapis.com/css2?family=Josefin+Sans:wght@600;700&display=swap');
       * {
           margin: 0;
           padding: 0;
           box-sizing: border-box;
-          font-family: 'Josefin Sans',  sans-serif;
+          font-family: 'Josefin Sans', 'Arial', 'Ubuntu',  sans-serif;
           font-style: normal;
+          font-weight:700;
+          text-align: center;
       }
-      .width-screen{
+      .board{
           min-width:1060px;
       }
-      .convert {
-          width: 127px;
-          height: 58px;
-          text-align: center;
+      .centre {
```

Comment:
this class work with margin only. So my proposition is to create your classes - 
"my-0" with styles "margin-top: 0; margin-bottom: 0;" and "mx-auto" with styles "margin-left: auto; margin-right: auto". This is how it works in Bootstrap